### ISSUE TITLE
Carbonite is absolutely not functional in Battlegrounds

### ISSUE THREAD

**Issue by TeskaTreska** [IrcDirk/Carbonite-Classic#390]
The map in bgs does not work at all. You can't see flags, occupied territory, you can't move with the map. The only thing you can see is where the players are moving, but you don't know anything about what's going on in the game. It hasn't been working for a long time, we were hoping that it would change with the last update, but still nothing works. Will work be done on the repair?

**Comment by IrcDirk** [comment]
Nope, changes in WOW API prevents getting more info from BGs/Dungeons :(
